website/docs/library/restable.md

```diff
@@ -1,25 +1,239 @@
+|    13 |      11,626,681,248,842 |   11,626,681,248,830 |                    12 |
+|    14 |      81,386,768,741,882 |   81,386,768,741,870 |                    12 |
+|    15 |     569,707,381,193,162 |  569,707,381,193,150 |                    12 |
+
+
+## Cell areas
+
+:::caution
+Cell areas are computed with a **spherical** model of the earth using the
+[authalic radius](https://en.wikipedia.org/wiki/Earth_radius#Authalic_radius)
+given by
+[WGS84](https://en.wikipedia.org/wiki/WGS84)/[EPSG:4326](https://epsg.io/4326).
+:::
+
+### Average area in km<sup>2</sup>
+
+The area of an H3 cell varies based on its position relative to the
+[icosahedron vertices](../core-library/overview.md).
+We show the **average** hexagon areas for each resolution.
+All pentagons within a resolution have the same area.
+
+
+|   Res |   Average <ins>Hexagon</ins> Area (km<sup>2</sup>) |   Pentagon Area* (km<sup>2</sup>) |   Ratio (P/H) |
+|------:|---------------------------------------------------:|----------------------------------:|--------------:|
+|     0 |                                4,357,449.416078381 |               2,562,182.162955496 |        0.5880 |
+|     1 |                                  609,788.441794133 |                 328,434.586246469 |        0.5386 |
+|     2 |                                   86,801.780398997 |                  44,930.898497879 |        0.5176 |
+|     3 |                                   12,393.434655088 |                   6,315.472267516 |        0.5096 |
+|     4 |                                    1,770.347654491 |                     896.582383141 |        0.5064 |
+|     5 |                                      252.903858182 |                     127.785583023 |        0.5053 |
+|     6 |                                       36.129062164 |                      18.238749548 |        0.5048 |
+|     7 |                                        5.161293360 |                       2.604669397 |        0.5047 |
+|     8 |                                        0.737327598 |                       0.372048038 |        0.5046 |
+|     9 |                                        0.105332513 |                       0.053147195 |        0.5046 |
+|    10 |                                        0.015047502 |                       0.007592318 |        0.5046 |
+|    11 |                                        0.002149643 |                       0.001084609 |        0.5046 |
+|    12 |                                        0.000307092 |                       0.000154944 |        0.5046 |
+|    13 |                                        0.000043870 |                       0.000022135 |        0.5046 |
+|    14 |                                        0.000006267 |                       0.000003162 |        0.5046 |
+|    15 |                                        0.000000895 |                       0.000000452 |        0.5046 |
+
+*: Within a given resolution, all pentagons have the same area.
```

Comment:
Not sure you need to restate this - I'd keep either here or above but doubt you need both.

Reply:
I don't see much cost to keeping both; it's only one line. And I don't see a quicker, cleaner way to explain the difference in table headers where hexagons get "Average" but pentagons don't (that also catches the use case of users jumping directly to the table they care about and ignoring everything else).

For the purpose of getting this diff landed, I'll keep it as is, but I also wouldn't mind if anyone put up an alternative proposal. :)